### DISCUSSION TITLE
How to choose comparative species？

### DISCUSSION THREAD

**Discussion by zhangwenda0518** [hahnlab/CAFE5#125] [Q&A] [answered]
Hello, author.
I have run successfully in the sample file and browsed through the manual, tutorial, but in my species analysis,  i still have a confusion, for my species, how can i choose species to analyze?
1. choose how many species is relatively appropriate;
2. What are the selection principles of these species? Is it based on species kinship?

I am studying a wetland plant, it has a strong adaptability, so when I choose species, is to choose similar characteristics of plants? Or choose plants that are not similar or even far apart?

Furthermore, for the interpretation of the results，what is the number of species shrinking or expanding at a node relative to the neighboring branches, or against the overall background, or against changes in the root ancestral species?

I think this is a very important question, can you give me some advice?

Best wish for you ！

**Comment by hahnlab-user** [comment, marked as answer]
Hi,

This is a very complex set of questions, to which there is no single answer. CAFE can analyze a very large number of species, so that should not be a barrier. The largest barrier is how deep the tree is, as ancestral state inferences too far in the past have a large amount of error associated with them. The general rule in CAFE is that lambda*t<1, where t is the time to the root and lambda is the rate of evolutionary change. 

I hope this helps you decide.


matt
